src/lib/ng-package/package.ts

```diff
@@ -88,3 +88,10 @@ export class NgPackage {
     return [this.primary, ...this.secondaries].find(entryPoint => entryPoint.moduleId === moduleId);
   }
 }
+
+export interface AssetEntry {
+  glob: string;
+  input: string;
+  ignore?: string[];
+  output: string;
+}
```

Comment:
This is actually redundant since we can use the types generated from the JSON schema.

```ts
import { AssetPattern } from '../../ng-package.schema';
```

```suggestion

```